### DISCUSSION TITLE
Can any one help me to setup this project using docker am facing some issue while setting up this project ?

### DISCUSSION THREAD

**Discussion by taruntailor7** [ToolJet/ToolJet#10224] [General]
Need help for setting up this project. using docker please react out to me here or on my mail : taruntailor7@gmail.com
Thank you.

**Comment by FurquanAnwer** [comment]
@taruntailor7 I have the same issue. Did you receive any help ? Can you help me out ?

**Reply by PriteshKiri** [reply]
Hey @FurquanAnwer,

If you could share the error logs, it would be greatly appreciated. This will help us debug the issue better. Thank you!

**Comment by PriteshKiri** [comment]
Hey @taruntailor7,

Could you please provide a bit more detail about the errors you're encountering with Docker? Sharing the error logs would be really helpful. Thanks!
